dist/index.html

```diff
@@ -9,8 +9,7 @@
 
   <body>
     <h1>Safety Dungeon Master Control Centre</h1>
-    <div id="firebaseui-auth-container"></div>
-    <div id="loader">Loading...</div>
+    <div id="app">Heck</div>
```

Comment:
This div is sort of a hook. Elsewhere, Vue will push new Vue-enabled HTML into this div

We should never actually see "Heck" but it might briefly be loaded? Or might appear if the app fails in certain ways?

Reply:
Yep. All correct.